docs/content/contributing/issues-and-pull-requests.md

```diff
@@ -35,8 +35,7 @@ The development workflow is the following:
    2. Run `git submodule update --init --recursive`
 2. Write the code that is needed :D
 3. Add integration tests if necessary
-4. Get the linters with `make install_linters` and install `jq` with the package manager of your OS
-5. Use `make clean all` to build image locally and run tests (note that tests work on Linux **only**)
+4. [Prepare your environment and run linting and tests](./tests.md)
```

Comment:
```suggestion
4. [Prepare your environment and run linting and tests][docs-tests]
```

And add `[docs-tests]: ./test.md` to the bottom of the document, next to `docs-environment`. Makes it a little easier for us to maintain.

Also, I noticed the numbered list needs to be updated since you deleted the 5th entry. This won't show up on the rendered docs but we should probably fix that here :)

The rendered docs are rendered wrong however. This isn't your fault, but the not so great markdown parser our docs use. I can see that indents are only 3 spaces instead of 4. You don't have to bother with this, especially if that doesn't fix the rendering properly (may also need a blank line before the nested lists start).